CHANGELOG.unreleased.md

```diff
@@ -34,5 +35,8 @@
 >
 > In case of conflict, the highest (lowest in previous list) `$version` wins.
 
+- @xen-orchestra/backups patch
+- @xen-orchestra/proxy patch
```

Comment:
`Proxy` ?
I am only seeing `@xen-orchestra/backups` and `@xen-orchestra/cr-seed-cli`.

Reply:
Yes, but `backups` is a library which impacts the `xo-server` and `proxy` apps, which therefore must also be released :slightly_smiling_face: